templates/home/dashboard.html

```diff
@@ -431,22 +431,19 @@ <h4 class="text-center mt-3">Volunteers Present</h4>
 
 <script src="https://cdn.jsdelivr.net/npm/js-cookie@rc/dist/js.cookie.min.js"></script>
 <script>
-    function hw_done(id, checkbox_id) {
-        var checked = false;
+    function hw_done(student_id, checkbox_id) {
         $('#' + checkbox_id).change(function(){
-            if($(this).prop("checked") == true){
-               checked = true;
-            } else {
-                checked = false;
-            }
-            console.log(checkbox_id, id, checked)
+            var date = new Date($('#class-date').val()).toUTCString();
```

Comment:
You can directly send the `{{ selected_date }}` context value here.

Reply:
I tried it, it didn't worked

Reply:
It should. Just put `var date = "{{ selected_date }}";`